target/linux/pistachio/base-files/etc/diag.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+#
+# Copyright (C) 2017 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+. /lib/functions/leds.sh
+. /lib/pistachio.sh
+
+status_led="marduk:red:heartbeat"
+
+set_state() {
+
+	case "$1" in
+	preinit)
+		status_led_blink_preinit
+		;;
+	failsafe)
+		status_led_blink_failsafe
+		;;
+	preinit_regular)
+		status_led_blink_preinit_regular
+		;;
+	upgrade)
+		status_led_blink_slow
+		;;
+	done)
+		status_led_set_heartbeat
```

Comment:
Can you use here `status_led_set_on` as in (almost) all other targets?

In fact, [I was going to drop both](https://git.lede-project.org/?p=lede/pepe2k/staging.git;a=log;h=refs/heads/20170309_failsafe-firstboot) `status_led_set_heartbeat` and `status_led_blink_slow` functions as the first one is used only in ar7 target and second one seems to be there since introduction in 2013 and... has never been used (based on what I see in history).